api/handler/atttributes_test.go

```diff
@@ -1,13 +1,10 @@
 package handler
```

Comment:
Just noticed a typo in the filename `api/handler/a_ttt_ributes_test.go` :)

Reply:
Sorry, I don't understand